databuilder/query_language.py

```diff
@@ -535,7 +536,9 @@ def _wrap(qm_node):
         ]
         assert len(matches) == 1
         cls = matches[0]
-    return cls(qm_node)
+        wrapped = cls(qm_node)
+        wrapped._type = type_
```

Comment:
It's a bit confusing having both `_type` and `type_` floating around.  Is it worth standardising?  Or is there some meaning that I'm missing -- and if so, could we have a comment at the top of the module?

Reply:
Ah OK, that sounds pretty standard.  If there's not project-specific semantics here, then it's probably not worth documenting.